Assets/Scripts/SethRefactor/FoodSpawner.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ScavengerWorld
+{
+    public enum FoodDistribution { Random, PsuedoUniform, Clumped }
+
+    public class FoodSpawner : MonoBehaviour
+    {
+        [Tooltip("Method of food distribution across the arena.")]
+        [SerializeField] private FoodDistribution Distribution = FoodDistribution.Random;
+        
+        [Range(0, 500)]
+        [Tooltip("Number of food items to generate at the beginning of every episode (max 500).")]
+        [SerializeField] private int NumFood = 20;
+        
+        [Range(5, 200)]
+        [SerializeField] private float XRange = 20;
+        
+        [Range(5, 200)]
+        [SerializeField] private float ZRange = 20;
+
+        [SerializeField] private Gatherable foodPrefab;
+
+        private HaltonSequence sequencer;
+        private List<Gatherable> gatherables = new();
```

Comment:
What does this do?

Reply:
Is this a new C# cool quick init thing?

Reply:
Yeah with the latest C# version you can just new() and the compiler will know what type you're trying to create